docs/source/scripts.rst

```diff
@@ -48,5 +48,8 @@ ivadomed_download_data
 
 .. autofunction:: ivadomed.scripts.download_data.install_data
 
+ivadomed_visualize_and_compare_testing_models
+"""""""""""""""""""""""""""""""""""""""""""""
 
+.. autofunction:: ivadomed.scripts.ivadomed_visualize_and_compare_testing_models
```

Comment:
```suggestion
.. autofunction:: ivadomed.scripts.visualize_and_compare_testing_models
```

You will also append the name of the major function of this script, see examples above.